src/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -159,16 +176,22 @@ public with sharing class AFFL_MultiRecordTypeMapper {
     * @return String The name of the key affiliation Contact field.
     ********************************************************************************************************/
 	public String getKeyAfflFieldByAccRecordType(ID recordTypeId) {
-		for(String recTypeName : accRecTypeToContactLabel.keySet()) {
-            if (Schema.Sobjecttype.Account.getRecordTypeInfosByName().get(recTypeName) != null)
-            {
+		for (String recTypeName : accRecTypeToContactLabel.keySet()) {
```

Comment:
the spacing is off here, in vs code looks like these are tabs and not spaces?

Reply:
Fixed it.